nuxeo-drive-client/nxdrive/manager.py

```diff
@@ -6,6 +6,10 @@
 import sys
 import logging
 from urlparse import urlparse
+import urllib2
+import js2py.pyjs
```

Comment:
Initially I tried` includes ` array option and still the build did not load pypac library properly. This option `import js2py.pyjs` worked well so I did not pursue the `includes` array option. I will try again.